### PR TITLE
fix(meetings): fix intermittent auto leave meeting issue

### PR DESCRIPTION
**Pull request by rarajes2**
Fixes the intermittent issue where a user was auto-leaving the meeting even without joining it.

JIRA - [SPARK-379330](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-379330)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,8 +30,8 @@
   "dependencies": {
     "webex": "^2.54.0",
     "@webex/component-adapter-interfaces": "^1.30.5",
-    "@webex/components": "^1.272.2",
-    "@webex/sdk-component-adapter": "^1.112.0"
+    "@webex/components": "^1.272.3",
```
Comment (Neeraj-swarnkar) [inline package.json:33]: @rarajes2, why we don't have package-lock.json changes as part of upgrade dependencies

**Review by Neeraj-swarnkar — COMMENTED** [review]
Added comments..

File: package.json (hunk 1 above)
Comment (rarajes2) [reply package.json:33]: Added `package-lock.json` file. Thanks

**Review by Neeraj-swarnkar — APPROVED** [review]
Looks Good..

**Pull request merged**